src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/ControllerAdvice.kt

```diff
@@ -0,0 +1,136 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config
+
+import org.slf4j.LoggerFactory
```

Comment:
Controller advice - to catch expected exception and translate them into ResponseEntity<ErrorMessage>.